src/server/qgsrequest.h

```diff
@@ -0,0 +1,47 @@
+/***************************************************************************
```

Comment:
I don't understand the need for this class. Or better, I think we should eventually create a QgsRequest and QgsReponse classes for the server refactoring but this is another story.